packages/platform_maps_flutter/LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright (c) 2019, Luis Thein
```

Comment:
Is this till the case? Are we not taking over the copyright?

Reply:
You definitely have my permission to change it 😄

Reply:
I'll check with legal :)

Reply:
It would become:

```suggestion
Copyright (c) 2024, Albert Heijn Technology
Copyright (c) 2019, Luis Thein
```